cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -78,6 +79,40 @@ def issue_jwt_token_with_jwk(token_body, file_name, algorithm = Algorithms::RS25
     )
   end
 
+  def issue_jwt_token_with_x5c(token_body, algorithm = Algorithms::RS256)
+    cert = self_signed_certificate(rsa_keys[@default_file_name])
+
+    @jwt_token = JWT.encode(
+      token_body,
+      rsa_keys[@default_file_name],
+      algorithm,
+      {
+        kid:  base64_x5t_from_certificate(cert),
+        x5c:  cert.to_pem.gsub("\n", ""),
```

Comment:
Are you sure that `x5c` value is aligned with [RFC](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.6).

Reply:
As far as i know, yes, `x5c` should be the pem of the certificate that signed the token

Reply:
> The "x5c" (X.509 certificate chain) Header Parameter contains the X.509 public key certificate or certificate chain [RFC5280] corresponding to the key used to digitally sign the JWS.  The certificate or certificate chain is represented as a JSON array of certificate value strings.  Each string in the array is a base64-encoded (Section 4 of [RFC4648] -- not base64url-encoded) DER [ITU.X690.2008] PKIX certificate value.  The certificate containing the public key corresponding to the key used to digitally sign the JWS MUST be the first certificate.  This MAY be followed by additional certificates, with each subsequent certificate being the one used to certify the previous one.  The recipient MUST validate the certificate chain according to RFC 5280 [RFC5280] and consider the certificate or certificate chain to be invalid if any validation failure occurs.  Use of this Header Parameter is OPTIONAL.
See Appendix B for an example "x5c" value.

Reply:
`The "x5c" (X.509 certificate chain) Header Parameter contains the X.509 public key certificate or certificate chain [RFC5280] corresponding to the key used to digitally sign the JWS`

If im not wrong, thats what i did

Reply:
Why do you stop reading after the first sentence :-)?

Reply:
> The certificate or certificate chain is represented as a JSON array of certificate value strings. Each string in the array is a base64-encoded (Section 4 of [RFC4648] -- not base64url-encoded) DER [ITU.X690.2008] PKIX certificate value.